x/incentives/keeper/distribute.go

```diff
@@ -44,7 +45,8 @@ func (k Keeper) AllocateAcrossGauges(ctx sdk.Context, activeGroups []types.Group
 	for _, group := range activeGroups {
 		err := k.syncGroupWeights(ctx, group)
 		if err != nil {
-			ctx.Logger().Error("error syncing group gauge weights, skipping", "group gauge id", group.GroupGaugeId, "error", err.Error())
+			telemetry.IncrCounter(1, types.SyncGroupGaugeFailureTelemetryName)
+			ctx.Logger().Error(types.SyncGroupGaugeFailureTelemetryName, "group gauge id", group.GroupGaugeId, "error", err.Error())
```

Comment:
@alessandrolomanto 

2. Let's make an alert for this counter please `incentives_group_gauge_sync_failure`. No dashboards needed

Reply:
Could you pls edit the string `group gauge id` to `group_gauge_id`?

Reply:
What values can the counter take: 0 or 1? Should the alert be triggered when it differs from 0?
For this case I prefer to create alert based on logs.
Eg: Error syncing group gauge with id: < group_gauge_id>

Reply:
If it increases, we should get alerted